src/components/App.js

```diff
@@ -41,6 +62,54 @@ class App extends React.Component {
     this.setState({ filters });
   };
 
+  createNewBook = () => {
+    this.setState({ bookFormContent: {} });
+    this.setState({ editing: false });
+    this.setState({ showEditForm: true });
+  };
+
+  loadBook = isbn => {
+    const books = this.state.books;
+    const index = books.findIndex(b => b.isbn === isbn);
+    const bookFormContent = books[index];
```

Comment:
These two lines could use `find` instead of `findIndex`. 
```suggestion
    const bookFormContent = books.find(b => b.isbn === isbn);
```